src/lib/components/Editor/ContentPreviewContainer.tsx

```diff
@@ -123,6 +123,7 @@ const ContentPreviewContainer: React.FC<IContentPreviewContainerProps> = ({
 
   const cardPreviewData: IPost = {
     author: {
+      id: user ? user.id : 0,
```

Comment:
This will be implemented correctly after merge this [PR](https://github.com/ita-social-projects/dokazovi-fe/pull/225)